service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/startup.sh

```diff
@@ -191,9 +201,11 @@ trap 'exit_handler $? $LINENO $BASH_COMMAND' EXIT
 # Let the UI know the script has started
 set_guest_attributes "${STATUS_ATTRIBUTE}" "STARTED"
 
-# Add login user to sudoers
-emit "Adding login user to sudoers"
+# Add login user to sudoers and enable sudo without password on it
+emit "Adding login user to sudoers..."
+readonly NO_PROMPT_SUDOERS_FILE="/etc/sudoers.d/no-sudo-password-prompt-${LOGIN_USER}"
 sudo usermod -aG sudo $LOGIN_USER
```

Comment:
Drop the first "sudo" here as it isn't needed (and update the LOGIN_USER variable reference):

```
usermod -aG sudo "${LOGIN_USER}"
```